css/about.css

```diff
@@ -138,18 +138,58 @@ nav a {
 }
 
 .card {
+    width: 100%;
+}
+
+figure {
     width: 50%;
     margin: 10px;
     padding-bottom: 25px;
-    border-bottom: 1px dashed black;
+    /* border-bottom: 1px dashed black; */
     font-size: var(--font-size);
+    -webkit-transform: perspective(500);
```

Comment:
Sorry, i am unable to seem that animation on my side, but the code are right, keep going